x-pack/plugins/data_usage/server/services/index.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { AppContextService } from './app_context';
+import { AutoOpsAPIService } from './autoops_api';
+import type { DataUsageContext } from '../types';
+import { MetricTypes } from '../../common/rest_types';
+
+export class DataUsageService {
+  private appContextService: AppContextService;
+  private autoOpsAPIService: AutoOpsAPIService;
+
+  constructor(dataUsageContext: DataUsageContext) {
+    this.appContextService = new AppContextService(dataUsageContext);
+    this.autoOpsAPIService = new AutoOpsAPIService(this.appContextService);
+  }
+
+  getLogger(routeName: string) {
+    return this.appContextService.getLogger().get(routeName);
+  }
+  async getMetrics({
+    from,
+    to,
+    metricTypes,
+    dataStreams,
+  }: {
+    from: string;
+    to: string;
```

Comment:
I'm hoping they come through with strings for date ranges. I don't see the [open API spec](https://github.com/elastic/autoops-services/blob/master/monitoring/service/specs/serverless_project_metrics_api.yaml) updated yet about it. 🤞🏼